README.md

```diff
@@ -103,10 +103,10 @@ Follow these steps to below to get the library up and running locally.
 
 ### Setup
 
-- Install [Node.js](https://nodejs.org) version 12
+- Install [Node.js](https://nodejs.org) version 16
```

Comment:
Updating node version to 16 as stated in the package.json https://github.com/MetaMask/design-tokens/blob/0910f70eb36b2bc1e9f40569d648ac524e23485e/package.json#L88